### ISSUE TITLE
no Go files in /go/src/github.com/mongodb/vault-plugin-secrets-mongodbatlas

### ISSUE THREAD

**Issue by heprotecbuthealsoattac** [mongodb/vault-plugin-secrets-mongodbatlas#25]
Hi, any hints how to fix the thing below?
```
~ # go get github.com/mongodb/vault-plugin-secrets-mongodbatlas
package github.com/mongodb/vault-plugin-secrets-mongodbatlas: no Go files in /go/src/github.com/mongodb/vault-plugin-secrets-mongodbatlas
```

**Comment by heprotecbuthealsoattac** [comment]
If I run:
```
go get -v github.com/mongodb/vault-plugin-secrets-mongodbatlas/...
```
it produces the binary that I'm after but returns non-zer exit code.

**Comment by themantissa** [comment]
@heprotecbuthealsoattac  wanted to let you know this work is now included in the 1.4Beta1 version of Vault released by HashiCorp and will be in the GA 1.4 Vault version.  I'll close this PR as you won't need to build this on your own now.  Thank you!
